lib/helpers/path-to-eslint.js

```diff
@@ -0,0 +1,20 @@
+/*eslint-env node*/
+
+'use strict';
+
+var join = require('path').join;
+var asyncExec = require('./async-exec');
+
+/**
+ * Get the path to the ESLint executable in the dependencies
+ *
+ * @returns {Promise->String} the path to the ESLint executable
+ */
+module.exports = function() {
+  return asyncExec('npm', ['bin'])
+    .then(function(location) {
+      return join(
+        location.trim(), '../ember-cli-eslint/node_modules/.bin/eslint'
```

Comment:
Yeah, I made that comment looking for help before I remembered `require.resolve`.  That's exactly what I'm doing now.

https://github.com/alexlafroscia/ember-cli-eslint/blob/0fcab0ade6480f47abdba5c17fee7e9c16a09e4d/lib/helpers/eslint-command.js#L8